### PR TITLE
Remove Hacktoberfest mention from PR template

### LINKED ISSUE #5273: Update PR template
### 📋 Explain your issue

Since Hacktoberfest is over, shouldn't we update the template and remove all the info regarding this event?

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Removed comments of hacktoberfest from PR Template

#### Related issues & labels (optional)

- Closes #5273 
- Suggested label: consistency/formatting

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 <!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌 -->
 
 #### Description (required)
-<!-- Please describe the change you are proposing, and why -->
```
Comment (at-the-vr) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: I don't think this portion should be deleted 🤔

Comment (dreyfus92) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: That part wasn't deleted, it is still there 😁

Comment (at-the-vr) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: Ohh I saw the Deletion (Red background thing) and completely missed the Green background 🤦

**Review by at-the-vr — APPROVED** [review]
Looks solid 👍

**Review by delucis — APPROVED** [review]
Thanks for taking care of this! 🧹

**Pull request merged**